apps/bmd/src/AppRoot.tsx

```diff
@@ -460,16 +469,20 @@ class AppRoot extends React.Component<Props, State> {
     const election = await this.props.card.readLongObject<Election>()
     /* istanbul ignore else */
     if (election) {
-      this.setState({ election }, this.resetTally)
-      this.storeElection(election)
+      const electionDefinition = {
+        election,
+        electionHash: sha256(JSON.stringify(election)),
```

Comment:
This isn't guaranteed to produce the right hash. We need to compute the hash at the time we import the election, from a string. Unfortunately, that means digging farther into the depths of the `Card` API. Instead of `readLongObject` we should be doing `readLongString` or something.